jwst/flatfield/flat_field.py

```diff
@@ -1524,11 +1524,9 @@ def interpolate_flat(image_flat, image_dq, image_err, image_wl, wl):
             # dq and err arrays are the same size, so treat them the same
             return image_flat.reshape((ysize, xsize)), image_dq, image_err
         else:
-            return (
-                image_flat.reshape((ysize, xsize)),
-                image_dq.reshape((ysize, xsize)),
-                image_err.reshape((ysize, xsize))
-            )
+            return (image_flat.reshape((ysize, xsize)),
+                    image_dq.reshape((ysize, xsize)),
+                    image_err.reshape((ysize, xsize)))
```

Comment:
I think original code was OK both from PEP8 and just visual point of view.

Reply:
I think either is fine.  In this instance, I prefer my edit.  I like fewer lines of code and I don't think it's necessary here to have the beginning and ending parenthesis on their own line.